File: src/org/cappuccino_project/ide/intellij/plugin/stubs/interfaces/ObjJAccessorPropertyStub.java
package org.cappuccino_project.ide.intellij.plugin.stubs.interfaces;

import org.cappuccino_project.ide.intellij.plugin.psi.impl.ObjJAccessorPropertyImpl;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;


public interface ObjJAccessorPropertyStub extends ObjJMethodHeaderDeclarationStub<ObjJAccessorPropertyImpl> {
    @NotNull
    String getContainingClass();
    @Nullable
    String getVariableName();
    @Nullable
    String getGetter();
    @Nullable
    String getSetter();
    @Nullable
    String getVarType();
}

File: src/cappuccino/ide/intellij/plugin/stubs/interfaces/ObjJAccessorPropertyStub.kt
package cappuccino.ide.intellij.plugin.stubs.interfaces

import cappuccino.ide.intellij.plugin.psi.impl.ObjJAccessorPropertyImpl


interface ObjJAccessorPropertyStub : ObjJMethodHeaderDeclarationStub<ObjJAccessorPropertyImpl>, ObjJResolveableStub<ObjJAccessorPropertyImpl> {
    val containingClass: String
    val variableName: String?
    val getter: String?
    val setter: String?
    val varType: String?
}
